### PR TITLE
BM-412 Fix ie 10 bug.

### PR DESCRIPTION
**Pull request by greg-fti**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -60,8 +60,9 @@
   },
   "dependencies": {
     "axios": "^0.18.0",
+    "babel-polyfill": "^6.26.0",
     "core-js": "^2.5.5",
-    "fast-xml-parser": "^3.9.10",
+    "fast-xml-parser": "~3.0.0",
```
Comment (EyesOnlyNet) [inline package.json]: how bad, that we have to roll back the version of the xml-parser.
did you double checked, if the options of the parser doesn't changed between these versions?

File: src/UbpCrsAdapter.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
+import 'babel-polyfill';
 import 'polyfills';
```
Comment (EyesOnlyNet) [inline src/UbpCrsAdapter.js]: do we still need the "polyfills" file if we utilize "babel-polyfill"?

File: src/UbpCrsAdapter.js [hunk 3]
```diff
@@ -1,3 +1,4 @@
+import 'babel-polyfill';
```
Comment (EyesOnlyNet) [inline src/UbpCrsAdapter.js]: is this import still needed when we have the "babel-polyfill" in the webpack-"entry"-option?
i guess, it should be bundled automatically with the adapter then.

File: package.json (hunk 1 above)
Comment (greg-fti) [reply package.json]: I found the other way of solving the problem. Please look at newer commit.

**Pull request merged**